docs/testplan/dual_tor/dual_tor_orch_test_plan.md

```diff
@@ -175,3 +175,19 @@ ip route add 1.1.1.1 nexthop via 10.0.0.57 nexthop via 10.0.0.59 nexthop via 10.
     | Repeat PTF tests as above | SLB  | Verify packet forwarding based on mux state|
     ||||
     | Verify teardown by shutting peering session one by one | SLB  | After one session is down, verify other peering session is active and routes present|
+
+1. Standalone tunnel route
+    This test is to verify standalone tunnel route is added properly when there is a `FAILED` or `INCOMPLETE` neighbor entry.
+    * Active-Standby DualToR
+
+    | Step | Goal | Expected results |
+    |-|-|-|
+    | Flush neighbor entry on `standby` side with `garp_service` on and ToR to server link down. (Note that missing neighbor entry scenario has been verified by case above, this is to mimic a `FAILED` neighbor entry) | Forwarding through tunnel | Verify tunnel-route on standby ToR | 
```

Comment:
If we want to mimic a `FAILED` entry, why does `garp_service` need to be on, and why do we take the link down? We can leave `garp_service` stopped and just `ping` some IP in the VLAN subnet that ICMP is not configured for.

Reply:
We can simply test this by sending some downstream traffic to a IP in the vlan subnet(neither server nor soc address), and check if there will be any tunnel traffic as a result.